tests/engine/test_detokenize.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from transformers import AutoTokenizer
+
+from vllm.transformers_utils.tokenizer import detokenize_incrementally
+
+
+@pytest.fixture(scope="module")
+def llama_tokenizer():
+    return AutoTokenizer.from_pretrained("hf-internal-testing/llama-tokenizer")
```

Comment:
While this doesn't have to be in this PR, can we make this test other tokenizers as well? E.g., IIRC, Code-Llama has a different tokenizer.